composer.json

```diff
@@ -44,9 +44,13 @@
         "drupal/view_unpublished": "1.x-dev",
         "drush/drush": "^9.2",
         "oomphinc/composer-installers-extender": "^1.1",
+        "symfony/debug": "^3.3.3",
+        "symfony/filesystem": "^3.4",
+        "symfony/finder": "^3.4",
         "zaporylie/composer-drupal-optimizations": "^1.0"
     },
     "require-dev": {
+        "drupal/console": "^1.9",
```

Comment:
I'm also torn if this should be a dev dependency as Drush is not. :thinking:

Reply:
I could see moving drush there - it's available on Acquia and I'm guessing most servers we would deploy to. If not you could always move it into require for that specific project. Alternatively if console was moved it shouldn't have any negative impact on the server environment short of space.